src/libbson/fuzz/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+if (MONGO_FUZZ)
+
+add_executable(fuzz_test_init_from_json
+   fuzz_test_init_from_json.c)
```

```suggestion
add_executable(fuzz_test_init_from_json EXCLUDE_FROM_ALL fuzz_test_init_from_json.c)
```
Do not build fuzz executable as part of build-and-install routines.

---

Done